src/main/java/com/mcmoddev/bot/command/CommandXY.java

```diff
@@ -0,0 +1,34 @@
+package com.mcmoddev.bot.command;
+
+import com.mcmoddev.bot.util.Utilities;
+import sx.blah.discord.handle.obj.IMessage;
+import sx.blah.discord.util.EmbedBuilder;
+
+public class CommandXY implements Command
+{
+    @Override
+    public void processCommand(IMessage message, String[] params)
+    {
+        final StringBuilder builder = new StringBuilder();
+        final EmbedBuilder embed = new EmbedBuilder();
+
+        builder.append(
+                "The XY problem is asking about your attempted solution rather than your actual problem." +
+                "This leads to enormous amounts of wasted time and energy, both on the part of people asking for help, and on the part of those providing help.\n" +
```

Comment:
Utilities.SEPERATOR does the same thing as \n\r

From the looks of this, there would be a 2 line space.

Reply:
The `\n` bit is likely not intentional. It is not platform specific, and is a bad habit to use.